binding/lua/docs/API.md

```diff
@@ -0,0 +1,102 @@
+# Multiverso Torch Binding API
```

Comment:
Does lua have anything like python doc string?
How about integrating the API doc in the code?